nilearn/maskers/tests/test_base_masker.py

```diff
@@ -31,7 +31,7 @@
 if compare_version(sklearn_version, "<", "1.5.0"):
     extra_valid_checks.append("check_estimator_sparse_data")
 
-if compare_version(sklearn_version, ">", "1.6"):
+if compare_version(sklearn_version, ">", "1.5.2"):
```

Comment:
Could it be `sklearn_version, ">=", "1.6"` instead?

Reply:
oops sorry, merged before refreshing the page

I think I tried this but you get into some 'weird' things with dev / pre release versions:

```
In [9]: compare_version("1.6.0", ">=", "1.6")
Out[9]: True

In [10]: compare_version("1.6.dev0", ">=", "1.6")
Out[10]: False
```

that's why things were working... until 1.6 was out....

probably we could improve `compare_version`